core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -527,8 +527,11 @@ public synchronized void losePrimacy() {
 
   @Override
   public synchronized Map<String, Long> getCurrentSequenceNumbers() {
-    long currentGlobalState = mStateMachine.getLastAppliedSequenceNumber();
     Map<String, Long> sequenceMap = new HashMap<>();
+    if (mStateMachine == null) {
```

Comment:
We shouldn't return empty sequences. Please replace this with below:

`Preconditions.checkState(mStateMachine != 1, "State machine not initialized");
`

Reply:
then is this more like a bug and check why `mStateMachine` is `null`?
@bzheng888  have you seen this in your environment that `mStateMachine` is `null`?

Reply:
> then is this more like a bug and check why `mStateMachine` is `null`?
> @bzheng888 have you seen this in your environment that `mStateMachine` is `null`?
Yeah, I debugged in my environment and it's really a must present bug, I think the registerMetrics() in the construct method  is useless, the metrics is never taken effect.

Reply:
Yeah, I debugged in my environment and it's really a must present bug, I think the registerMetrics() in the construct method is useless, the metrics is never taken effect.

Reply:
@bzheng888 So would you like to move the `registerMetrics` to the end of `initServer` ?